core/src/test/scala/org/broadinstitute/dsde/rawls/spendreporting/SpendReportingServiceSpec.scala

```diff
@@ -870,7 +872,7 @@ class SpendReportingServiceSpec extends AnyFlatSpecLike with Matchers with Mocki
     val billingProjectSpendExport =
       BillingProjectSpendExport(RawlsBillingProjectName(""), RawlsBillingAccountName(""), None)
     doReturn(Future.successful(billingProjectSpendExport)).when(service).getSpendExportConfiguration(any())
-    doReturn(Future.successful(TestData.googleProjectsToWorkspaceNames))
+    doReturn(Future.successful(TestData.billingProjectsToWorkspaces))
```

Comment:
here, and in a few other pre-existing unit tests, the mock was wrong; it was returning the incorrect datatype. I've updated all the cases I saw to be correct.